decidim-elections/app/views/decidim/elections/admin/proposals_imports/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= decidim_form_for(@form, url: election_question_proposals_import_path, html: { class: "form import_proposals" }) do |f| %>
+  <% if @form.origin_components.any? %>
+    <div class="card">
+      <div class="card-divider">
+        <h2 class="card-title"><%= title %></h2>
+      </div>
+
+      <div class="card-section">
+        <div class="row column">
+          <%= f.select :origin_component_id, @form.origin_components_collection, prompt: t(".select_component") %>
+        </div>
+        <div class="row column">
+          <%= f.check_box :import_all_accepted_proposals %>
+        </div>
+      </div>
+    </div>
+
+    <div class="button--double form-general-submit">
+      <%= f.submit t(".create") %>
+    </div>
+  <% else %>
+      <p><%= t(".no_components") %></p>
```

Comment:
```suggestion
    <p><%= t(".no_components") %></p>
```

Reply:
Thanks for spotting this! I updated the code. 😊